### DISCUSSION TITLE
Beta4 Upload very temperamental

### DISCUSSION THREAD

**Discussion by TechplexEngineer** [forkineye/ESPixelStick#440] [Q&A]
Hello, huge thanks to @forkineye and the contributors 4.x with FPP support is am amazing accomplishment.

I've just updated my ESPixelStick v3 to use the 4.x Beta4 firmware and saw some odd behavior.

1. When uploading sequences from xlights (v39) using FPP connect the web interface on the ESPixelStick sometimes shows the websocket disconnected spinner.
2. When uploading sequences from xlights sometimes the ESPixelStick web interface becomes unresponsive and seems to require a reboot. And the xlights upload fails with a CURL error about the server not sending a response. Or a different CURL response 56 recv failure

![image](https://user-images.githubusercontent.com/175873/145138452-d72c50e8-7e3f-4972-9aac-09e5eb9a06f8.png)

I tried the upload a number of times without success, and after a number of reboots the upload succeeded and its now working flawlessly.

Just wanted to see if others are seeing this odd behavior or its just me?

**Comment by forkineye** [comment]
There have been some people having issues when uploading multiple files. It's best to not have a page open to the controller as well when uploading. There's an issue tracking upload problems here - https://github.com/forkineye/ESPixelStick/issues/427
